rtil/src/native/character.rs

```diff
@@ -91,6 +91,19 @@ impl AMyCharacter {
         assert!(!ptr.is_null());
         unsafe { (*ptr).steamid }
     }
+
+    pub fn movement_mode(&mut self) -> u8 {
+        self.movement_mut().movement_mode
+    }
+    pub fn set_movement_mode(&mut self, value: u8) {
+        self.movement_mut().movement_mode = value;
+    }
+    pub fn forward_backward_fly_speed(&mut self) -> f32 {
+        self.movement_mut().max_fly_speed
+    }
+    pub fn set_forward_backward_fly_speed(&mut self, value: f32) {
+        self.movement_mut().max_fly_speed = value;
+    }
```

Comment:
Why did you rename `max_fly_speed` to `forward_backward_fly_speed`? I think keeping with the UE naming helps others understand the code easier.